superset/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -6,7 +6,7 @@ import * as Actions from '../actions';
 import SqlEditor from './SqlEditor';
 import CopyQueryTabUrl from './CopyQueryTabUrl';
 import { areArraysShallowEqual } from '../../reduxUtils';
-import { getParamFromQuery } from '../../../utils/common';
+import URI from 'urijs';
```

Comment:
the sql lab entry js file 13.2mb 🙈 
![screenshot 2017-04-03 17 46 22](https://cloud.githubusercontent.com/assets/130878/24637381/8f04fcc4-1895-11e7-9f6a-cde4a3afa018.png)